applications/tari_base_node/src/main.rs

```diff
@@ -27,6 +27,7 @@
 #![deny(unused_must_use)]
 #![deny(unreachable_patterns)]
 #![deny(unknown_lints)]
+#![allow(dead_code)]
```

Comment:
Is this leftover from debugging?

Reply:
errm, no idea how that got in there.... good catch